### PR TITLE
[Add] Support lumi.sen_ill.agl01 GZCGQ11LM

### LINKED ISSUE #1220: Support lumi.sen_ill.agl01 (GZCGQ11LM)
I have a GZCGQ11LM light sensor that has been paired with the gateway(lumi.gateway.mgl03) successfully, however the device doesn't show up in the list. 

I tried search all the logs, there is nothing error etc.
the only logline looks relevant is:

```
2023-11-25 17:33:22 [D] 192.168.1.xxx [BASE] 0x54ef4410008ed7d6 (0x2a4e) setup zigbee (lumi.sen_ill.agl01):
```

Could anybody help? Thanks

### PR DESCRIPTION
**Pull request by ileodo**
Support lumi.sen_ill.agl01 GZCGQ11LM, which can close #1220

### CODE REVIEW

**Comment by AlexxIT** [comment]
You can add it as second model to existing device

**Comment by ileodo** [comment]
thanks for the response. is there an example I can follow?

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/XiaomiGateway3/pull/1231/files#diff-6d3a357986914979e62209fc11210e277cecca4932c612dc515b9694df80197aR604-R606

File: custom_components/xiaomi_gateway3/core/converters/devices.py [hunk 1]
```diff
@@ -504,6 +504,13 @@
...
 DEVICES += [{
+    "lumi.sen_ill.agl01": ["Aqara", "Light Sensor CN", "GZCGQ11LM"],
+    # "support": 5,  # @AlexxIT bug with gw fw 1.5.1
+    "spec": [
+        Converter("illuminance", "sensor", mi="2.p.1"),
+        BatteryConv("battery", "sensor", mi="3.p.1"),  # voltage, mV
+    ],
+}, {
     "lumi.sen_ill.mgl01": ["Xiaomi", "Light Sensor EU", "GZCGQ01LM"],
     # "support": 5,  # @AlexxIT bug with gw fw 1.5.1
```
Comment (ileodo) [inline custom_components/xiaomi_gateway3/core/converters/devices.py:515]:
```suggestion
    "lumi.sen_ill.mgl01": ["Xiaomi", "Light Sensor EU", "GZCGQ01LM"],
    # "support": 5,  # @AlexxIT bug with gw fw 1.5.1
    "lumi.sen_ill.agl01": ["Aqara", "Light Sensor CN", "GZCGQ11LM"],
```

**Comment by ileodo** [comment]
thanks, updated!

**Comment by AlexxIT** [comment]
Thanks!

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/XiaomiGateway3/releases/tag/v3.3.5

**Pull request merged**